pages/admin/permit-holder/[permitHolderId].tsx

```diff
@@ -30,84 +30,106 @@ import {
   UpdateApplicantResponse,
   UPDATE_APPLICANT_MUTATION,
 } from '@tools/pages/admin/permit-holders/update-applicant'; // Update applicant types
+import {
+  UpdateMedicalInformationRequest,
+  UpdateMedicalInformationResponse,
+  UPDATE_MEDICAL_INFORMATION_MUTATION,
+} from '@tools/pages/admin/permit-holders/update-medical-information';
 
 type Props = {
   readonly permitHolderId: number;
 };
 
+// Must be defined outside of PermitHolder function otherwise variable does not maintain value
+let physicianId: number;
```

Comment:
+1 with what Oustan said at standup, would be great to find a different way to get around this